packages/wallet/src/test-utils.ts

```diff
@@ -1,3 +1,7 @@
 export * from './test-utils/generateTestWallet';
 export * from './test-utils/seedTestWallet';
 export * from './test-utils/launchNode';
+export * from './test-utils/launch-custom-provider-and-get-wallets';
+export { WalletConfig } from './test-utils/wallet-config';
+export { AssetId } from './test-utils/asset-id';
+export { launchTestNode as launchNode } from '@fuel-ts/providers/test-utils';
```

Comment:
I reexported it here so that the PR doesn't introduce breaking changes. I'd remove it in another PR after others are made known of the `TestNodeLauncher`'s existence.